README.md

```diff
@@ -113,6 +114,10 @@ We have standardized on a few abbreviations that are allowed to use:
 - `ui` for user interface
 - `util` or `utils` for utility or utilities, as in `StringUtils`
 
+> <p id="cta"> Call-to-action (CTA) buttons are the buttons you use in your website and on your
+> landing pages to guide users towards your goal conversion. It's the part of the landing page that
+> the user needs to click in order to take the action you want them to take.</p>
```

Comment:
> buttons are the buttons

can't we just do "_A **Call to action** is usually a button that is used..._" ?

> your website and on your landing pages

the landing page is part of your website? can't we simply state website?

> goal conversion
what's "goal conversion" ?

> It's the part of the landing page that the user needs to click in order to take the action you want them to take.
maybe "_it's used to guide the user towards taking some action e.g. a call to the latest session of the event_"